index.js

```diff
@@ -111,7 +111,8 @@ function createMainWindow() {
 		win.maximize();
 	}
 
-	win.webContents.loadURL(config.get("url"));
+	const urlToLoad = config.get("resumeOnStart") ? config.get("url") : "https://music.youtube.com";
```

Comment:
nit: root url could be imported from default config

Reply:
Small typo: option is named `resumeSongOnStart` in other places and not `resumeOnStart`